client/api_client.go

```diff
@@ -96,6 +99,9 @@ type Option struct {
 	UseWebSocketConn bool
 	// ExpireSeconds indicates the number of seconds after which the authentication of the request sent to the SP will become invalid，the default value is 1000.
 	ExpireSeconds uint64
+	// ForceToUseSpecifiedSpEndpointForDownloadOnly indicates a fixed endpoint of sp and shall only be used by get object cmd.
+	// Any other api shall not use this parameter.
```

Comment:
Cmd should not be mentioned  here which has nothing to do with this project.  
They should be independent .  Need concretize this meaning , for example , “indicates a fixed SP endpoint to which to send the download  request，If this option is set, the client can only make download requests, and can only download from the fixed endpoint. ”

Reply:
@flywukong Make sense, thanks for pointing out. Fixed comments in client & options.